isthmus/build.gradle.kts

```diff
@@ -108,6 +108,8 @@ tasks {
     archiveBaseName.set("isthmus")
     manifest { attributes(mapOf("Main-Class" to "io.substrait.isthmus.PlanEntryPoint")) }
```

Comment:
I just noticed that we reference PlanEntryPoint here even though that was renamed to `IsthmusEntryPoint` as part of https://github.com/substrait-io/substrait-java/pull/209/files and now that lives in `isthmus-cli` after https://github.com/substrait-io/substrait-java/pull/249/files.

I'm not a Gradle expert so I'm not sure what this is for, but I suspect it's not load-bearing 😅 

In any case, not something we need to fix as part of this change, just noting it down.